apps/frontend/src/app/5_pages/MarketMakingPage/components/AdjustAndDepositModal/AdjustAndDepositModal.tsx

```diff
@@ -110,15 +205,15 @@ export const AdjustAndDepositModal: FC<AdjustAndDepositModalProps> = ({
                 isInitialDeposit ? (
                   <div className="flex justify-end w-full">
                     <MaxButton
-                      value={maxBalance}
+                      value={fromWei(maxBalance)}
```

Comment:
```suggestion
                      value={maxBalance}
```
You won't need to convert the value if you adjust it as I told you.